index.html

```diff
@@ -416,65 +416,172 @@ <h2 id="liaisons">
+      Decision Process
+    </h2>
+    
+    <p>This group will seek to make decisions where there is consensus. 
+    Groups are free to decide how to make decisions (e.g. Participants 
+    who have earned Committer status for a history of useful contributions 
+    assess consensus, or the Chair assesses consensus, or where consensus 
+    isn't clear there is a Call for Consensus [CfC] to allow multi-day online 
+    feedback for a proposed course of action). It is expected that participants 
+    can earn Committer status through a history of valuable contributions as is 
+    common in open source projects. After discussion and due consideration of 
+    different opinions, a decision should be publicly recorded (where GitHub is 
+    used as the resolution of an Issue).
+    </p>
+    
+    <p>
+      If substantial disagreement remains (e.g. the group is divided) 
+      and the group needs to decide an Issue in order to continue to make progress, 
+      the Committers will choose an alternative that had substantial support 
+      (with a vote of Committers if necessary). Individuals who disagree 
+      with the choice are strongly encouraged to take ownership 
+      of their objection by taking ownership of an alternative fork. 
+      This is explicitly allowed (and preferred to blocking progress) 
+      with a goal of letting implementation experience inform which 
+      spec is ultimately chosen by the group to move ahead with.
+    </p>
+    
+    <p>
+      Any decisions reached at any meeting are tentative and should be 
+      recorded in a GitHub Issue for groups that use GitHub and otherwise 
+      on the group's public mail list. Any group participant may object to 
+      a decision reached at an online or in-person meeting within 7 days of 
+      publication of the decision provided that they include clear technical 
+      reasons for their objection. The Chairs will facilitate discussion to try 
+      to resolve the objection according to the <a href="#decision">decision 
+      process</a>.
+    </p>
+    
+    <p>
+      It is the Chairs' responsibility to ensure that the decision process is fair,
+      respects the consensus of the CG, and does not unreasonably favour or 
+      discriminate against any group participant or their employer.
+    </p>
+
+    
     <h2 id="chairs">
       Chair Selection
     </h2>
+
     <p>
       Participants in this group choose their Chair(s) and can replace their
       Chair(s) at any time using whatever means they prefer. However, if 5
       participants, no two from the same organisation, call for an election,
       the group must use the following process to replace any current Chair(s)
       with a new Chair, consulting the Community Development Lead on election
-      operations (e.g., voting infrastructure and using RFC 2777).
+      operations (e.g., voting infrastructure and using 
+      <a href="https://tools.ietf.org/html/rfc2777">RFC 2777</a>).
```

Comment:
This is a specification for an unbiased voting algorithm, it's not a voting tool or voting process.  What is the Community Group expected to do with this specification?